arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -2642,6 +2889,196 @@
+        }
+      },
+      "description": "The response of the List rule operation."
+    },
+    "RuleCreateOrUpdateParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Rule name."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RuleProperties"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate operation."
+    },
+    "RuleResource": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RuleProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of Rule Resource."
+    },
+    "Filter": {
+      "type": "object",
+      "properties": {
+         "RequiresPreprocessing": {
+          "type": "boolean",
+          "description": "Preprocesses the filter expression and returns a preprocessed FilterExpression."
+        },
+		"SqlFilter": {
+          "$ref": "#/definitions/SqlFilter",
+          "description": "Represents a SQL filter which is a composition of an expression and an action that is executed in the pub/sub pipeline."
+        },
+		"CorrelationFilter": {
+          "$ref": "#/definitions/CorrelationFilter",
+          "description": "Represents the correlation filter expression."
+        }
+      },
+      "description": "MessageDescribes a filter expression that is evaluated against a BrokeredMessage."
+    },
+	"SqlFilter": {
+      "type": "object",
+      "properties": {
+         "SqlExpression": {
+          "type": "string",
+          "description": "The SQL expression."
+        },
+		"RequiresPreprocessing": {
+          "type": "boolean",
+          "description": "Gets a value indicating whether the SQL filter expression requires preprocessing."
+        },
+		"CompatibilityLevel": {
+		"format": "int32",
+		"type": "integer",
+          "description": "This property is reserved for future use. An integer value showing the compatibility level, currently hard-coded to 20."
+		}
+      },
+      "description": "Represents a filter which is a composition of an expression and an action that is executed in the pub/sub pipeline."
+    },	
+	"CorrelationFilter": {
+      "type": "object",
+      "properties": {
+         "CorrelationId": {
+          "type": "string",
+          "description": "Gets the identifier of the correlation."
+        },
+		"MessageId": {
+          "type": "string",
+          "description": "Gets the identifier of the message."
+        },		
+		"To": {
+          "type": "string",
+          "description": "Gets the address to send to."
```

Comment:
should this be an email address or twitter handle? Can you provide an example in the description?